codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -197,24 +206,21 @@ solidLoop :: HasCallStack => [Point] -> Picture
 solidLoop ps = SolidClosedCurve callStack ps
 
 {-# WARNING solidLoop "Please use solidClosedCurve instead of solidLoop." #-}
+rectangleToPolygon :: CallStack -> Double -> Double -> Picture
```

Comment:
I'll disagree slightly with Gabe's comment.  The helper function I think you want here is this:

    rectangleVertices :: Double -> Double -> [Point]
    rectangleVertices w h = [ (w / 2, h / 2), (w / 2, -h / 2), (-w / 2, -h / 2), (-w / 2, h / 2) ]

I believe you want this in CodeWorld.Driver.  Then you can do something like this:

    pictureToDrawing (Rectangle _ w h) = Shape $ pathDrawer (rectangleVertices w h) 0 True False

and something similar for the other three constructors.